airflow/models/dag.py

```diff
@@ -1662,6 +1662,7 @@ def run(
         verbose=False,
         conf=None,
         rerun_failed_tasks=False,
+        rerun_succeeded_tasks=False,
```

Comment:
You should add a test for the change here

Reply:
I have add a test_cli_backfill_rerun_succeeded_tasks func in test_dag_command.py，does it meet your requirements?

Reply:
Good. You also need to add this kind of test :
https://github.com/apache/airflow/blob/1e1b212417e089b43474bc19eec05b654afe98fb/tests/jobs/test_backfill_job.py#L574-L576
for the rerun success task

Reply:
OK, I have added the test in recent commit (a85e664)